src/main/java/org/sagebionetworks/bridge/services/ParticipantVersionService.java

```diff
@@ -0,0 +1,168 @@
+package org.sagebionetworks.bridge.services;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.dao.ParticipantVersionDao;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.models.accounts.Account;
+import org.sagebionetworks.bridge.models.accounts.ParticipantVersion;
+import org.sagebionetworks.bridge.models.accounts.SharingScope;
+import org.sagebionetworks.bridge.models.studies.Enrollment;
+import org.sagebionetworks.bridge.time.DateUtils;
+
+@Component
+public class ParticipantVersionService {
+    private ConsentService consentService;
+    private ParticipantVersionDao participantVersionDao;
+
+    @Autowired
+    public final void setConsentService(ConsentService consentService) {
+        this.consentService = consentService;
+    }
+
+    @Autowired
+    public final void setParticipantVersionDao(ParticipantVersionDao participantVersionDao) {
+        this.participantVersionDao = participantVersionDao;
+    }
+
+    /** Creates a participant version from an account. */
+    public void createParticipantVersionFromAccount(Account account) {
+        if (!account.getRoles().isEmpty()) {
+            // Accounts that have roles aren't research participants. Don't export them to Synapse.
+            return;
+        }
+        if (account.getSharingScope() == SharingScope.NO_SHARING) {
+            // no_sharing means we don't export this to Synapse, which means we can skip making a Participant Version.
+            return;
+        }
+        Optional<Boolean> isConsented = consentService.isConsented(account);
+        if (!isConsented.isPresent() || !isConsented.get()) {
+            // If participant is not consented, don't create a participant version.
+            // If we're not sure whether they're consented, assume that they are not.
+            return;
+        }
```

Comment:
Because most studies don't include signing a consent, I think you'll need to check and see if the user is enrolled. Their records should be exported and made available to everyone who has access to the studies the user is enrolled in. For example in MTB, no participant would pass this test, I don't think.

Reply:
Dwayne read this comment and agrees to switch to Enrollment instead.